lib/nexthop_group.c

```diff
@@ -1130,6 +1176,13 @@ static int nexthop_group_write(struct vty *vty)
 
 		vty_out(vty, "nexthop-group %s\n", nhgc->name);
 
+		if (nhgc->nhg.nhgr.buckets)
+			vty_out(vty,
+				" resilient buckets %u idle_timer %u unbalanced_timer %u\n",
```

Comment:
as Trey said, here _ should be changed to - too?

Reply:
yep fixed